app/views/_cascade/blocks/html/_footer.html

```diff
@@ -43,14 +43,70 @@
+                <span class="sr-only">youTube</span>
+                <svg class="icon icon-youtube" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 32 32">
+                  <path d="M31.7 9.6c0 0-0.3-2.2-1.3-3.2-1.2-1.3-2.6-1.3-3.2-1.4-4.5-0.3-11.2-0.3-11.2-0.3h0c0 0-6.7 0-11.2 0.3-0.6 0.1-2 0.1-3.2 1.4-1 1-1.3 3.2-1.3 3.2s-0.3 2.6-0.3 5.2v2.4c0 2.6 0.3 5.2 0.3 5.2s0.3 2.2 1.3 3.2c1.2 1.3 2.8 1.2 3.5 1.4 2.6 0.2 10.9 0.3 10.9 0.3s6.7 0 11.2-0.3c0.6-0.1 2-0.1 3.2-1.4 1-1 1.3-3.2 1.3-3.2s0.3-2.6 0.3-5.2v-2.4c0-2.6-0.3-5.2-0.3-5.2zM12.7 20.2v-9l8.6 4.5-8.6 4.5z"></path>
+                </svg>
+              </a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/company/10701?trk=tyah" target="_blank">
+                <span class="sr-only">LinkedIn</span>
+                <svg class="icon icon-linkedin2" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 32 32">
+                  <path d="M12 12h5.5v2.8h0.1c0.8-1.4 2.7-2.8 5.5-2.8 5.8 0 6.9 3.6 6.9 8.4v9.6h-5.8v-8.5c0-2 0-4.7-3-4.7-3 0-3.5 2.2-3.5 4.5v8.7h-5.8v-18z"></path><path d="M2 12h6v18h-6v-18z"></path><path d="M8 7c0 1.7-1.3 3-3 3s-3-1.3-3-3c0-1.7 1.3-3 3-3s3 1.3 3 3z"></path>
+                </svg>
+              </a>
+            </li>
+            <li>
+              <a href="https://www.snapchat.com/add/chapmanu" target="_blank">
+                <span class="sr-only">Snapchat</span>
+                <svg class="icon icon-snapchat" height="24" viewBox="0 0 32 32" width="24" xlink="http://www.w3.org/2000/svg">
+                  <path d="M16.5,1.4c3.4,0,6.3,1.9,7.7,4.9c0.5,0.9,0.5,2.4,0.5,3.4c0,1.2-0.1,2.4-0.2,3.6c0.2,0.1,0.4,0.1,0.5,0.1c0.6,0,1.3-0.5,1.9-0.5c0.6,0,1.6,0.5,1.6,1.2c0,1.8-3.8,1.5-3.8,3.1c0,0.3,0.1,0.5,0.2,0.8c0.9,2,2.6,3.9,4.6,4.7c0.5,0.2,1,0.4,1.5,0.5c0.4,0.1,0.5,0.4,0.5,0.6c0,1.3-3.3,1.8-4.2,1.9c-0.4,0.5-0.1,2-1.1,2c-0.8,0-1.6-0.3-2.4-0.3c-0.4,0-0.8,0-1.2,0.1c-2.2,0.5-3,3-6.7,3c-3.6,0-4.5-2.5-6.7-2.8c-0.4-0.1-0.8-0.1-1.2-0.1c-0.8,0-1.6,0.3-2.4,0.3c-1.1,0-0.7-1.4-1.1-2c-0.9-0.1-4.2-0.6-4.2-1.9c0-0.4,0.2-0.5,0.5-0.6c0.5-0.1,1-0.2,1.6-0.5c2-0.8,3.8-2.7,4.6-4.7c0.1-0.3,0.2-0.5,0.2-0.8c0-1.6-3.8-1.3-3.8-3.1c0-0.7,0.9-1.2,1.5-1.2c0.5,0,1.2,0.5,1.9,0.5c0.2,0,0.5,0,0.6-0.1C7.7,12.2,7.6,11,7.6,9.8c0-1,0.1-2.5,0.5-3.4C9.8,2.7,12.6,1.4,16.5,1.4z"></path>
+                </svg>
+              </a>
+            </li>
+            <li>
+              <a href="https://itunes.apple.com/us/institution/chapman-university/id430678922" target="_blank">
+                <span class="sr-only">iTunes</span>
+                <svg class="icon icon-itunes" height="24" viewBox="0 0 32 32" width="24" xlink="http://www.w3.org/2000/svg">
+                  <path d="M30,0h2v23c0,2.8-3.1,5-7,5s-7-2.2-7-5s3.1-5,7-5c2,0,3.7,0.6,5,1.5V8l-16,3.6V27c0,2.8-3.1,5-7,5s-7-2.2-7-5s3.1-5,7-5c2,0,3.7,0.6,5,1.5V4L30,0z"></path>
+                </svg>
+              </a>
+            </li>
```

Comment:
What is the `xmlns`/`xlink` attribute? Is it necessary, I noticed they're all the same.

Reply:
It's a requirement for `image/svg` files, but I just checked right now and it works fine without it. I also confirmed it with a quick search:

- https://stackoverflow.com/a/34249810

It's not a requirement for inlined `svg` which I hadn't realized before. Thanks for the catch! I'll remove it in the next commit. We might want to also do the same for the newsroom footer, which is where I got these `svg`'s from.

Reply:
No, do not remove it. It's an explicit way to specify how the page should read that element. Without it, sometimes the page won't render the SVG at all. I've seen it with the OmniNav as a Service that I'm working on right now. It's much better practice to leave it.